test/e2e/specs/editor-tracking/editor-tracking__block-inserted-event.ts

```diff
@@ -217,13 +217,12 @@ describe(
 					templatePartBlock = new TemplatePartBlock( page, block );
 				} );
 
-				it( '"wpcom_block_inserted" event fires with "entity_context" === "template"', async function () {
+				it( '"wpcom_block_inserted" event fires', async function () {
 					const eventDidFire = await editorTracksEventManager.didEventFire(
 						'wpcom_block_inserted',
 						{
 							matchingProperties: {
 								block_name: 'core/template-part',
-								entity_context: 'template',
```

Comment:
This property does not appear to exist on this event anymore.

Reply:
I took a look and it seems the entity_context is a quick fix. It seems to be a result of a element ID change in the editor. https://github.com/Automattic/wp-calypso/pull/69184 should resolve the issue.